show/main.py

```diff
@@ -458,6 +490,23 @@ def expected(interfacename):
 
     click.echo(tabulate(body, header))
 
+# 'summary' subcommand ("show interfaces summary")
+@interfaces.command()
+@click.argument('interfacename', required=False)
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def summary(interfacename, verbose):
```

Comment:
This command was intentionally removed. Did you mean to add it back, or was your code out-of-date?